server/zoom/jwt.go

```diff
@@ -54,6 +54,16 @@ func (c *JWTClient) GetUser(user *model.User) (*User, *AuthError) {
 	return &zoomUser, nil
 }
 
+func (c *JWTClient) CreateMeeting(userEmail string) (*Meeting, error) {
+	var ret Meeting
+	meetingRequest := CreateMeetingRequest{
+		Topic: "Meeting created on Mattermost",
+		Type:  1,
```

Comment:
Can we have a constant for this, and also a `NewCreateMeetingRequest` function?

Reply:
The constant will be done. Regarding the function, I am not sure how useful will be. We may eventually add more different kinds of meetings, and this approach seems good for the time being. Also, this definition is only repeated on the jwt and oauth code, but the jwt code will be removed for the next release.